### ISSUE TITLE
UnhandledPromiseRejectionWarning: Error: regex is required

### ISSUE THREAD

**Issue by DeanKamali** [nknapp/node-child-service#3]
I'm running a binary that doesn't display any output when it runs, I'm getting the following exception 

```
(node:4476) UnhandledPromiseRejectionWarning: Error: regex is required
    at waitForMatch (C:\Users\user\Documents\bridge\node_modules\child-service\src\lib\wait-for-match.js:12:9)
    at ChildService.start (C:\Users\user\Documents\bridge\node_modules\child-service\src\index.js:65:4)
    at async IpcMainImpl.<anonymous> (C:\Users\deank\Documents\bridge\main.js:137:6)
```

**Comment by nknapp** [comment]
The main purpose, when I wrote this library, was to wait until the service is up and running.Of course, we could make it optional.

**Comment by nknapp** [comment]
I fixed this in 2.4.1, hope it's not too late for you...
